data/pages.yaml

```diff
@@ -155,24 +155,21 @@ community:
   title: Join the Ansible community
   intro: Open-source and collaboration are at the heart of the Ansible community, helping more people experience the power of automation and work better and faster together.
   collaborate:
-    title: Collaborate with the community
-    fork: Fork the Ansible repository
+    title: Collaborate
+    conduct: Review the Ansible Code of Conduct
+    help: Explore ways you can help
     ecosystem: Find an Ansible project
-    help: Explore other ways you can help
-  discuss:
-    title: Participate in community discussions
-    email: Get in touch by email
-    chat: Find us on Matrix
-    irc: Use the IRC bridge on Libera Chat
-  join:
-    title: Get involved
-    conduct: Read the code of conduct
-    working_groups: Take part in a working group
+  communicate:
+    title: Communicate
```

Comment:
```suggestion
    title: Join the conversation
```

Reply:
Join the conversation feels a little friendlier and more human than simply "communicate".

Reply:
Agreed